dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/encryption/AuditCrypto.java

```diff
@@ -2012,51 +2046,17 @@ static final byte[] decrypt(byte[] mesg, byte[] key, String cipher, boolean useJ
         }
 
         byte[] tmpMesg = null;
-        if (useJCE) {
+        if (fips140_3Enabled) {
```

Comment:
This appears to be a behavior change.  We always went down this path before because useJCE was always true.

Reply:
Yes, we use to have if (useJCE || fips140_3Enabled) and remove useJCE .. fixed it.

Reply:
Maybe you need tp refresh your browser?

Reply:
No I refreshed.  I see that `encrypt` was updated, but `decrypt` still has the wrong logic.